twitter_radix/twitter_radix/layouts/auth.py

```diff
@@ -0,0 +1,44 @@
+"""Shared auth layout."""
+import reflex as rx
+
+from ..components import container
+
+
+def auth_layout(*args):
+    """The shared layout for the login and sign up pages."""
+    return rx.box(
+        container(
+            rx.heading(
+                rx.chakra.span("Welcome to PySocial!"),
```

Comment:
what's the replacement in 0.4.0 for this?

Reply:
removed

Reply:
changed to text(..., as_"span")